src/native/corehost/build.sh

```diff
@@ -117,3 +107,8 @@ check_prereqs
 
 # Build the installer native components.
 build_native "$__TargetOS" "$__BuildArch" "$__scriptpath" "$__IntermediatesDir" "$__CMakeArgs" "installer component"
+
+if [[ "$__RuntimeFlavor" != "Mono" ]]; then
+    echo Copying "$__CoreClrArtifacts/corehost/."  to "$__CMakeBinDir/corehost"
+    cp -a "$__CoreClrArtifacts/corehost/."  "$__CMakeBinDir/corehost"
```

Comment:
Why do we  copy all files from corehost on Unix while on Windows, we copy the .exe and .pdb explicitly?

Reply:
We need to copy the entire "/corehost". There are only two files there on Windows - `singlefilehost.exe` and `PDB/singlefilehost.pdb`

Unix has a convenient command for copying the content of the entire `/corehost` and cmd alternative `xcopy` is harder to use - it gets confused whether we are copying a file or a folder, whether the destination has to exist or must not exist, and so on.

`copy` with individual 2 files ended up a lot more reliable.